collections/ansible_collections/cloudkit/templates/roles/ocp_virt_vm/tasks/create.yaml

```diff
@@ -127,44 +127,32 @@
       status: "True"
     wait_timeout: 600
 
-- name: Create VM service
-  kubernetes.core.k8s:
-    definition:
-      apiVersion: v1
-      kind: Service
-      metadata:
-        name: "{{ vm_name }}-service"
-        namespace: "{{ vm_target_namespace }}"
-        labels: "{{ default_vm_labels }}"
-      spec:
-        type: "{{ vm_service_type }}"
-        selector: "{{ default_vm_labels }}"
-        ports: "{{ vm_service_ports }}"
-    state: present
-  when: vm_expose_service and vm_service_ports | length > 0
-
 - name: Create a floating IP to attach to VM
   ansible.builtin.include_role:
     name: massopencloud.esi.floating_ip
   vars:
     floating_ip_state: present
     floating_ip_name: "{{ vm_name }}-floating-ip"
     floating_ip_network: external
+    floating_ip_properties:
+      vm_name: "{{ vm_name }}"
+      vm_namespace: "{{ vm_target_namespace }}"
```

Comment:
Maybe just call it `vm` instead of `vm_name`.

I would say we also want `instance` instead of `vm_namespace`. `instance` refers to the particular installation of O-SAC; for example, right now in hypershift1 we have the `ajamais` instance and the `tzumainn` instance, as well as `innabox-isaiah` and `innabox-lars`. The goal is to distinguish between these instances.

I'm also not sure we want to set `instance` to `vm_target_namespace`, which I *think* is a new namespace specifically created to hold VM resources. We would want something like `vm_order.metadata.namespace`. If that works, then in your testing you should see a floating IP with a tag of `instance_ajamais`